Library/Homebrew/cmd/audit.rb

```diff
@@ -878,12 +878,12 @@ def audit_line(line, lineno)
         problem "`#{$1}` is now unnecessary"
       end
 
-      if line =~ /#\{share\}\/#{formula.name}/
+      if line =~ %r{#\{share\}/#{formula.name}[/'"]}
         problem "Use \#{pkgshare} instead of \#{share}/#{formula.name}"
       end
 
-      if line =~ /share\/"#{formula.name}"/
-        problem "Use pkgshare instead of (share/\"#{formula.name}\")"
```

Comment:
The `()` wrapping here is intentional FWIW. It's intended to tell people to do:

```ruby
pkgshare.install "ducks"
```
rather than misleading them into doing:
```ruby
(pkgshare).install "ducks"
```
Since this one was written to primarily catch:
```ruby
(share/"name").install "ducks"
```
Style syntax.

Was this causing misleading audit comments on the style to use?